### PR TITLE
Create multiple  yaml/python output files for ndrop.F90/ccncalc

### PR DESCRIPTION
**Pull request by brigaudet**
1)  Create yaml/python output files over multiple timesteps for ndrop.F90/ccncalc (single vertical level and horizontal column)

2)  Create yaml/python output files for ndrop.F90/dropmixnuc over additional timesteps (all vertical levels for single horizontal column), and add additional module variables to input files.

### CODE REVIEW

File: components/eam/src/chemistry/yaml/cam_ndrop/f90_yaml/ccncalc_beg_yml.f90 [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        call write_var(unit_input,unit_output,'surften',surften)
+        call write_var(unit_input,unit_output,'twothird',twothird)
+        call write_var(unit_input,unit_output,'sq2',sq2)
+        call write_var(unit_input,unit_output,'exp45logsig',exp45logsig)
+        call write_var(unit_input,unit_output,'psat',psat)
+        call write_var(unit_input,unit_output,'supersat',supersat)
+
+        call write_var(unit_input,unit_output,'ntot_amode',ntot_amode)
+
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/cam_ndrop/f90_yaml/ccncalc_beg_yml.f90:142]: May we add the following variables: 
`lspectype_amode`, `lmassptr_amode`, `specdens_amode`, `spechygro`, `voltonumbhi_amode`, `voltonumblo_amode`, `numptr_amode`, `nspec_amode`  ?

Note that these variables are saved for `dropmixnuc`.

**Review by odiazib — COMMENTED** [review]
It looks great!

**Comment by jeff-cohere** [comment]
Looks good to me, too. Just curious--Is there a reason we include a `dt` input that is set to `0`? Is this needed for the parameterization, or just something that we always get?

**Comment by singhbalwinder** [comment]
`dt` is a leftover from the initial design. We can remove it. I don't think any of the tests are using it.

**Comment by jeff-cohere** [comment]
> `dt` is a leftover from the initial design. We can remove it. I don't think any of the tests are using it.

Thanks, @singhbalwinder . Not a big deal, but if it's easy to remove in the code generator, maybe that can be done whenever it's convenient. It certainly doesn't have to be removed in this PR.

File: components/eam/src/chemistry/yaml/cam_ndrop/f90_yaml/ccncalc_beg_yml.f90 (hunk 1 above)
Comment (brigaudet) [reply components/eam/src/chemistry/yaml/cam_ndrop/f90_yaml/ccncalc_beg_yml.f90:142]: Sure, I've gone ahead and done that.  I had only included module variables explicitly used in ccncalc, but these are used in subroutines ultimately called from ccncalc.

**Review by jeff-cohere — APPROVED** [review]

**Review by odiazib — APPROVED** [review]

**Comment by brigaudet** [comment]
image
Longer tests all pass, will go ahead and merge.

**Pull request merged**